repos/system_upgrade/el7toel8/actors/restrictedpcisscanner/libraries/restrictedpcisscanner.py

```diff
@@ -46,22 +42,22 @@ def get_restricted_pcis(http):
     # TODO this constant should be above the scope of this func.
     #   However due to the way how actor tested with CurrentActorMocked
     #   we can't do this.
-    API_URL = "http://{host_port}/api/{api_version}/".format(
+    API_URL = "http://{host_port}/api/pes/".format(
```

Comment:
`host_port` variable naming is puzzling to be honest, it took some time to go from "why is it referencing the port only" to "oh, it means hostname_with_port".
Can we have good old `host` or `server_url` here?

Reply:
Sure, I'll change it.